apps/range-voting/app/script.js

```diff
@@ -38,8 +38,16 @@ async function handleEvents(response) {
     nextState = await startVote(nextState, response.returnValues)
     break
   case 'ExternalContract':
-    console.info('[RangeVoting > script]: received ExternalContract')
-    allocations = app.external(response.returnValues.addr, AllocationJSON.abi)
+    let funcSig = response.returnValues.funcSig
+    console.info('[RangeVoting > script]: received ExternalContract', funcSig)
+    // Should actually be a case-switch
+    if(funcSig == 'f1d12a23'){
+      console.log('Loading Projects Data')
+      resolve(loadVoteDataProjects(voteData, voteId))
+    } else {
+      console.log('Loading Allocations Contract')
+      allocations = app.external(response.returnValues.addr, AllocationJSON.abi)
```

Comment:
Is the range voting app only meant to be used with one allocations contract at a time?